### ISSUE TITLE
[Quest][Classic] Thunderfury Quest bugged

### ISSUE THREAD

**Issue by GPSRT** [dalaranwow/dalaran-wow#5447]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Upon killing Prince Thunderaan (14435), my dormant wind kissed blade (19018) was on his corpse, but I could not loot it.

**Expected behaviour**: Tell us what should happen instead
After killing Prince Thunderaan (14435), I should be able to loot the dormant wind kissed blade (19018).

**Steps to reproduce the problem**:

1. I cannot reproduce the problem because I cannot re-summon Prince Thunderaan (14435) since he spawns at the end of completing the quest with the bindings (7786) and that quest is complete.
2. 
3. 

**Include proofs for this behaviour**
I cannot produce proof because the NPC won't let me accept the quest to start over again, and he won't let me resummon Thunderaan.
You can verify Iceaxe has completed the bindings quest but has not yet picked up his wind kissed blade.

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
Quest Iceaxe has completed: 7786
Iceaxe has killed Prince Thunderaan: 14435
Item Iceaxe should have: 19018

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Rushor** [comment]
I'm not sure how this should behave. I don't think you should have the option to resummon him with a gossip option.

Please contact a gamemaster to let you complete the quest https://wowgaming.altervista.org/aowow/?quest=7786 again (autocomplete), in order to resummon Prince Thunderaan.
